src/client/pages/Login.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useState } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-unresolved
+import { FormularioInput } from '../components/FormularioInput';
+// import { EmailValidation, validarEm]ail } from '../helpers/validarCorreo';
+import usuarioContexto from '../state/state';
+import '../styles/registro.css';
+import * as EmailValidator from 'email-validator';
+
+export const Login = () => {
+  const [formData, setFormData] = useState({
+    email: '',
+    password: ''
+  });
+  const { user, setUser } = useContext(usuarioContexto);
+
+  const onChance = e => {
+    const { name, value } = e.target;
+    setFormData(prev => ({
```

Comment:
This creates an unecessary new object every time a value on the form changes. Consider have separate state for each field.